src/util.c

```diff
@@ -252,7 +252,7 @@ void print_test_results(struct ntttcp_test_endpoint *tep)
 	PRINT_INFO_FREE(log);
 	ASPRINTF(&log, "\t cycles/byte\t:%.2f", tepr->cycles_per_byte);
 	PRINT_INFO_FREE(log);
-	ASPRINTF(&log, "cpu busy (all)\t:%.2f%%", tepr->cpu_busy_percent * 100);
+	ASPRINTF(&log, "\t cpu busy (all)\t:%.2f%%", tepr->cpu_busy_percent * 100);
```

Comment:
Why include this change in this commit? It seems to be a separated topic.

Reply:
In order to align with the previous code.